app/policies/collection_policy.rb

```diff
@@ -16,6 +17,11 @@ def update?
     administrator? || manages_collection?(record)
   end
 
+  sig { returns(T::Boolean) }
+  def update_first_draft?
+    update? && record.head.first_draft?
+  end
```

Comment:
we are only allowed to edit a first draft of a collection if its actually a first draft -- this ensures we don't end up on the wrong edit page

Reply:
But we already have separate checks for that and redirect the user. So why do we need this?

Reply:
the redirects go the other way (ensuring you go to the first draft actions when needed), not ensuring you don't go to the first draft action when you shouldn't ... but that could be handled in the same way with a redirect for consistency

Reply:
Used redirects in the new controller and removed the policy changes.